codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/testutil/TestConfigCustomization.kt

```diff
@@ -22,7 +22,14 @@ fun stubConfigCustomization(name: String): ConfigCustomization {
         override fun section(section: ServiceConfig): Writable = writable {
             when (section) {
                 ServiceConfig.ConfigStruct -> rust("_$name: u64,")
-                ServiceConfig.ConfigImpl -> emptySection
+                ServiceConfig.ConfigImpl -> rust(
+                    """
+                    #[allow(missing_docs)]
```

Comment:
My bad. I forgot we have to escape the `#` in these.
```suggestion
                    ##[allow(missing_docs)]
```